environments/development/inventory.ini

```diff
@@ -98,3 +98,6 @@ swap_size=100M
 
 [control:vars]
 hostname='control'
+
+[django_manage:children]
+webworkers
```

Comment:
Would this be multiple machines if someone spun up webworkers?

Reply:
It would by my eye, which I'd suggest we get around by using `django_manage:0` below to pick off the first one.

Reply:
yeah, i'll add `django_manage:0` to the places i'm selecting them